apps/api/src/app.ts

```diff
@@ -46,8 +46,8 @@ app.register(oauthPlugin, {
   credentials: {
     auth: oauthPlugin.GOOGLE_CONFIGURATION,
     client: {
-      id: env.GOOGLE_CLIENT_ID,
-      secret: env.GOOGLE_CLIENT_SECRET,
+      id: env.GOOGLE_CLIENT_ID as string,
+      secret: env.GOOGLE_CLIENT_SECRET as string,
```

Comment:
why do you have to use `as` here? `env` is already giving this type defined right?